yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,129 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. Updates the
+          keyword on hardware and also updates on the cache, if cache
+          information is present in the config JSON.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory/hardware path of the FRU whose keyword needs to be
```

Comment:
Dbus/EEPROM - Will make it more readable?

Reply:
updated